### PR TITLE
Cameron's website design

### PR DESCRIPTION
**Pull request by jdfreder**

### CODE REVIEW

**Comment by jdfreder** [comment]
@ellisonbg @cameronoelsen

**Comment by rgbkrk** [comment]
:shipit:

**Comment by rgbkrk** [comment]
Uh.... Why was the CNAME deleted? We still need that.

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-[Jupyter.org](http://jupyter.org/) website files
+# jupytersite
```
Comment (rgbkrk) [inline README.md]: Should still link to jupyter.org for people stumbling on this repo.

**Comment by rgbkrk** [comment]
Looking good.

**Comment by Carreau** [comment]
Please don't name something `assets/>.svg` this is giving the stick to be beaten with.
Call-it explicitely like `chevron` or whatever, but not `>`.

**Comment by Carreau** [comment]
Are PNG's not transparent on purpose for apple icons ?

**Comment by Carreau** [comment]
Menu still need border bottom on mobile or it looks broken:
screen shot 2015-08-31 at 13 26 03

**Comment by Carreau** [comment]
bloomberg is in png and SVG, is it normal? and botch BnW and Color?

**Comment by Carreau** [comment]
padlock icon has the lock part leaking below the middle line:
screen shot 2015-08-31 at 13 28 28

**Comment by Carreau** [comment]
Nitpicking, but `circle[1-3]` was not obvious before actually looking at the image.

**Comment by Carreau** [comment]
Still old logo of continuum wit colors commited.

**Comment by Carreau** [comment]
Hub logo is still off centered:

screen shot 2015-08-31 at 13 34 58

Red boxes are same height.

**Comment by Carreau** [comment]
Nitpicking, Try to separate real assets that will be on page from the ones that are use to builds the one on page.

**Comment by Carreau** [comment]
"The kernel" , "Interactive Computing Protocol" and "Notebook DocumentFormat" text seem to be part of the icon, is that normal? 

The "Notebook DocumentFormat"  icon folded corner is weird, by being acute angle. Is it on purpose ?

**Comment by Carreau** [comment]
Personal perspective, the shadows on "Try" "Install" and "more" looks really out of place in the flat theme.
But that my point of view.

**Comment by Carreau** [comment]
`scroll.js` flood the console, remove the `console.log()`

**Comment by Carreau** [comment]
there si a miss-using of class and id in css, you have classes that are used only once, like `.front`,`.front2`,`.front3` and that have the same style copy and pasted two or three time. 

Plus some overridden bootstrap  theme:

```
.nav > li > a:hover {
    color: #E46E2E;
    background-color: transparent;
}
```

That probably should just be a rebuild of bootstrap with customized variables.

**Comment by Carreau** [comment]
I make a break in review ! Good job though looks really nice. 

+1 to move forward and refine on deployment.

**Comment by ellisonbg** [comment]
I am going to merge and work with Cameron to iterate today based on the feedback.

**Comment by rgbkrk** [comment]
:rocket: :fist:

**Comment by jdfreder** [comment]
Did @Carreau 's feedback get addressed?  Or do we need to make sure it's recorded as GitHub issues so it gets addressed eventually?

**Comment by ellisonbg** [comment]
I am not sure, @cameronoelsen can you check this?

On Tue, Sep 8, 2015 at 9:19 AM, Jonathan Frederic <notifications@github.com>
wrote:

> Did @Carreau <https://github.com/Carreau> 's feedback get addressed? Or
> do we need to make sure it's recorded as GitHub issues so it gets addressed
> eventually?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/jupyter.github.io/pull/15#issuecomment-138617878>
> .
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by cameronoelsen** [comment]
@Carreau @jdfreder @ellisonbg All of his changes were pushed last week :+1:

**Pull request merged**